oasislmf/pytools/getmodel/footprint.py

```diff
@@ -400,15 +400,33 @@ def get_event(self, event_id: int):
             df_footprint['from_intensity'] = df_footprint['from_intensity'].fillna(0)
 
             if len(df_footprint.index) > 0:
-                df_footprint['intensity_bin_id'] = np.floor(df_footprint.from_intensity + (
+                df_footprint['intensity'] = np.floor(df_footprint.from_intensity + (
                     (df_footprint.to_intensity - df_footprint.from_intensity) * df_footprint.interpolation))
-                df_footprint['intensity_bin_id'] = df_footprint['intensity_bin_id'].astype('int')
+                df_footprint['intensity'] = df_footprint['intensity'].astype('int')
+                intensity_bin_dict = pd.read_csv('static/intensity_bin_dict.csv')
```

Comment:
We have an abstraction to read what is in static. This should work and be in line with the rest of the code
intensity_bin_dict =  self.get_df_reader('intensity_bin_dict.csv').as_pandas()

Reply:
thanks I'll have a look